examples/slice-pizza/datadog-app/src/widget/SlicePizza/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { init, ModalSize } from '@datadog/ui-extensions-sdk';
+import { ChakraProvider, Container, Button, Image, Stack } from '@chakra-ui/react'
+
+import '@fontsource/tourney/variable.css'
+import '@fontsource/bebas-neue/index.css'
```

Comment:
Why didn't you use the google fonts and add them inside the index.html file?

Reply:
To be honest, I was just following the [Using Fonts](https://chakra-ui.com/docs/styled-system/recipes/using-fonts) guide from Chakra UI.